l10n/en-US/viewer.properties

```diff
@@ -251,21 +251,21 @@ printing_not_ready=Warning: The PDF is not fully loaded for printing.
 web_fonts_disabled=Web fonts are disabled: unable to use embedded PDF fonts.
 
 # Editor
-editor_free_text.title=Add FreeText Annotation
-editor_free_text_label=FreeText Annotation
-editor_ink.title=Add Ink Annotation
-editor_ink_label=Ink Annotation
+editor_text.title=Text
+editor_text_label=Text
+editor_draw.title=Draw
+editor_draw_label=Draw
 
-free_text_default_content=Enter text…
+text_default_content=Start typing…
 
 # Editor Parameters
-editor_free_text_color=Color
-editor_free_text_size=Size
-editor_ink_color=Color
-editor_ink_thickness=Thickness
-editor_ink_opacity=Opacity
+editor_text_color=Color
+editor_text_size=Size
+editor_draw_color=Color
+editor_draw_thickness=Thickness
+editor_draw_opacity=Opacity
```

Comment:
Here, and elsewhere, why are we changing the string-ids when the content doesn't actually change?
Won't this cause localizers to (unnecessarily) have to re-translate this?

Can we please just revert these changes?

Reply:
It was just to be consistent, i.e. avoid having some free_text and text.
I thought that adding a number was a bit ugly but if you're fine with that then I am too.